pandas/computation/tests/test_eval.py

```diff
@@ -1411,6 +1411,22 @@ def test_multi_line_expression_not_inplace(self):
         e = a + 2""", inplace=False)
         assert_frame_equal(expected, df)
 
+    def test_multi_line_expression_local_variable(self):
+        # GH 15342
+        tm.skip_if_no_ne('numexpr')
+        df = pd.DataFrame({'a': [1, 2, 3], 'b': [4, 5, 6]})
+        expected = df.copy()
+
+        local_var = 7
+        expected['c'] = expected['a'] * local_var
+        expected['d'] = expected['c'] + local_var
+        ans = df.eval("""
+        c = a * @local_var
+        d = c + @local_var
```

Comment:
This test does actually have a blank line.  The `""", inplace=...` that terminates the string causes a line of nothing but space at the end of the string.  Formulating the test case this way was how I found that the `.strip()` was needed in the compare.